app/i18n/locales/pt-BR.json

```diff
@@ -126,9 +126,9 @@
   "Cancel_editing": "Cancelar edição",
   "Cancel_recording": "Cancelar gravação",
   "Cancel": "Cancelar",
-  "changing_avatar": "trocando avatar",
-  "creating_channel": "criando canal",
-  "creating_invite": "criando convite",
+  "changing_avatar": "Trocando avatar",
```

Comment:
Not these kind of actions.
These are actions to be used in the middle of sentences, so they should be untouched.

The goal of the task is to change list items and titles.

Reply:
could you give me the name of the directory of the files that need to be changed ?

Reply:
@diegolmello